docs/concepts/glossary.md

```diff
@@ -88,6 +112,21 @@ A Daemon is a computer program that typically runs in the background. The IPFS d
 
 A Directed Acyclic Graph (DAG) is a computer science data structure adapted for use with versioned file systems, blockchains, and for modeling many different kinds of information. [IPLD](#ipld) data in IPFS is naturally a DAG. [More about DAG on Wikipedia](https://en.wikipedia.org/wiki/Directed_acyclic_graph).
 
+### DAG-JSON
+
+DAG-JSON is a [codec](#codec) that implements the [IPLD Data Model](https://ipld.io/glossary/#data-model) as JSON, plus some additional conventions for encoding links, which it does by claiming certain specific structures of map and assigning them this meaning. DAG-CBOR also adds a "link" type using a CBOR tag, to bring it in line with the IPLD Data Model. [More about DAG-JSON](https://ipld.io/docs/codecs/known/dag-json/)
+
+### DAG-JOSE
+
+[More in DAG-JOSE specification](https://ipld.io/specs/codecs/dag-jose/spec/)
```

Comment:
Can we add a short explanation of what DAG-JOSE is right here? This was taken from [the above link](https://ipld.io/specs/codecs/dag-jose/spec/):

```suggestion
A standard for signing and encrypting JSON objects. [More in DAG-JOSE specification](https://ipld.io/specs/codecs/dag-jose/spec/)
```

Don't know if that correctly summarizes that DAG-JOSE is/does.

Reply:
@warpfork @aschmahmann  @johnnymatthews  iiuc dag-jose is "ipld way of representing JOSE as CBOR" so maybe:
```suggestion
DAG-JOSE is a [codec](#codec) that defines CBOR serialization for JOSE, a standard for signing and encrypting JSON objects. [More in DAG-JOSE specification](https://ipld.io/specs/codecs/dag-jose/spec/)
```

Reply:
Nice. @lidel's suggestion looks good to me. But I'll wait for sign-off from Warp or Adin before merging.

Reply:
@lidel I think you should ditch "JSON objects" and just use "objects". This is one of the problems we have with dag-jose—it sounds like it's all about JSON, but it's not, you can encode and/or sign arbitrary data that conforms to the IPLD data model with it.

Also, if you can jam it in with finesse then the encryption can be either asymmetric or symmetric, so we have signing and both encryption forms.

Reply:
Ack. Let's go with simple "objects":

```suggestion
DAG-JOSE is a [codec](#codec) that defines CBOR serialization for JOSE, a standard for signing and encrypting objects. [More in DAG-JOSE specification](https://ipld.io/specs/codecs/dag-jose/spec/)
```

This is just a glossary, if reader cares about details they will click on "More" link.